### PR TITLE
Update LIT1799Lifeof.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Update, old record for a new work.

### CODE REVIEW

File: 1001-2000/LIT1799Lifeof.xml [hunk 1]
```diff
@@ -34,17 +33,64 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
       </encodingDesc>
       <profileDesc>
+        <abstract>
+            <p>A collection of homilies for each month of the year
+               describing the life and death of John the Baptist.
+                 It seems to be a rare composition, and it
+                  is different from a work of a similar structure
+                 contained in <ref type="mss" corresp="ESehpp007"/>, and another one
+                 in the edition
+                 <bibl><ptr target="bm:bm:Tarraqan2003GadlaYohannes"/></bibl>.</p>
```
Comment (eu-genia) [inline 1001-2000/LIT1799Lifeof.xml:44]: check the tag: only once bm:

File: 1001-2000/LIT1799Lifeof.xml [hunk 2]
```diff
@@ -34,17 +33,64 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+        </abstract>
          <langUsage><language ident="en">English</language></langUsage>
       </profileDesc>
+
       <revisionDesc>
          <change who="PL" when="2016-03-21"> Created file from google
             spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: text record</change>
       </revisionDesc>
```
Comment (eu-genia) [inline 1001-2000/LIT1799Lifeof.xml:53]: new change element

File: 1001-2000/LIT1799Lifeof.xml [hunk 3]
```diff
@@ -34,17 +33,64 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+        <abstract>
+            <p>A collection of homilies for each month of the year
+               describing the life and death of John the Baptist.
+                 It seems to be a rare composition, and it
+                  is different from a work of a similar structure
+                 contained in <ref type="mss" corresp="ESehpp007"/>, and another one
+                 in the edition
+                 <bibl><ptr target="bm:bm:Tarraqan2003GadlaYohannes"/></bibl>.</p>
+        </abstract>
          <langUsage><language ident="en">English</language></langUsage>
```
Comment (eu-genia) [inline 1001-2000/LIT1799Lifeof.xml:46]: add gez

File: 1001-2000/LIT1799Lifeof.xml [hunk 4]
```diff
@@ -1,11 +1,10 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT1799Lifeof">
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Life of John the Baptist</title>
-            <!-- apocryphon by Serapion of Thmuis -->
```
Comment (eu-genia) [inline 1001-2000/LIT1799Lifeof.xml:8]: Serapion is wrong? where is the info now?

File: 1001-2000/LIT1799Lifeof.xml [hunk 5]
```diff
@@ -34,17 +33,64 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                   ወከመዝ፡ ፈጸመ፡ ሠናየ፡ ሑረቱ፡ ብፁዕ፡ ወቅዱስ፡ ዮሐንስ፡ መጥምቅ፡ ወአውገዙ፡ መምሕራነ፡
+                   ቤተ፡ ክርስቲያን፡ ከመ፡ <pb n="78ra"></pb>
+                    ምግባራተ፡ ሥጋ፡ ወኢምንተኒ፡ በዕለተ፡ በዓሉ፡ ለቅዱስ፡
+                    ዮሐንስ፡ መጥመቅ፨ ጸሎቱ፡ ወበረከቱ፡ የሃሉ፡ ምስለ፡ አግብሪሁ፡ አቡነ፡ ኪዳነ፡
+                    ወልድ፡ ወለተ፡ ግብርኤል፡ ለዓ፡ ዓለ፡ አ።
+                 </ab>
+                 </div>
+         </div>
+         <div type="bibliography">
+         </div>
```
Comment (eu-genia) [inline 1001-2000/LIT1799Lifeof.xml:88]: why empty div?

File: new/LIT6166MiraclesJohnBaptist.xml [hunk 6]
```diff
@@ -53,10 +55,170 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+</div>
+              </div>
+              </div>
             <div type="bibliography">
                 <listRelation>
                     <relation name="lawd:hasAttestation" active="LIT6166MiraclesJohnBaptist" passive="ESmy018"/>
                     <relation name="lawd:hasAttestation" active="LIT6166MiraclesJohnBaptist" passive="ESmy023"/>
+                    <relation name="lawd:hasAttestation" active="LIT6166MiraclesJohnBaptist" passive="ESehpp007"/>
+                    <relation name="lawd:hasAttestation" active="LIT6166MiraclesJohnBaptist" passive="ESbgy012"/>
                 </listRelation>
```
Comment (eu-genia) [inline new/LIT6166MiraclesJohnBaptist.xml:222]: if there is another text/collection of miracles here you need a relation to state the differentFrom. 6165? nothing there...

****
If not already present, you also need, to describe the manuscripts, the overarching ID for the Hagiographic Dossier with Gadl, Miracles, Malk, etc. (cp. LIT1118DossierAntony, LIT5877DossierParaqlitos, etc.)

File: 1001-2000/LIT1799Lifeof.xml [hunk 7]
```diff
@@ -34,17 +33,64 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+         <div type="bibliography">
+             <listRelation>
+                 <relation name="saws:isRelatedTo" active="LIT1799Lifeof" passive="PRS5741Johnthe"/>
+               <relation name="lawd:hasAttestation" active="LIT1799Lifeof" passive="EAPrqm029"/>
+               <relation name="lawd:hasAttestation" active="LIT1799Lifeof" passive="ESmy023"/>
+
```
Comment (eu-genia) [inline 1001-2000/LIT1799Lifeof.xml:94]: you need relations for the different texts here, with their IDs

File: 1001-2000/LIT1799Lifeof.xml [hunk 8]
```diff
@@ -34,17 +33,64 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                    ዮሐንስ፡ መጥመቅ፨ ጸሎቱ፡ ወበረከቱ፡ የሃሉ፡ ምስለ፡ አግብሪሁ፡ አቡነ፡ ኪዳነ፡
+                    ወልድ፡ ወለተ፡ ግብርኤል፡ ለዓ፡ ዓለ፡ አ።
+                 </ab>
+                 </div>
+         </div>
+         <div type="bibliography">
+         </div>
+         <div type="bibliography">
+             <listRelation>
+                 <relation name="saws:isRelatedTo" active="LIT1799Lifeof" passive="PRS5741Johnthe"/>
```
Comment (eu-genia) [inline 1001-2000/LIT1799Lifeof.xml:91]: you need isAbout relation for John

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: 1001-2000/LIT1799Lifeof.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply 1001-2000/LIT1799Lifeof.xml:53]: ok

File: 1001-2000/LIT1799Lifeof.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply 1001-2000/LIT1799Lifeof.xml:44]: ok

File: 1001-2000/LIT1799Lifeof.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply 1001-2000/LIT1799Lifeof.xml:46]: ok

File: 1001-2000/LIT1799Lifeof.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply 1001-2000/LIT1799Lifeof.xml:8]: I removed it. This record was without one single relation to a manuscript. Yes, there is Serapion an alledged author of a Greek version, but is it attested in Ethiopic? I think we did not find it so far.  "The Acts of John by Serapion" would be just misleading a little bit, unless someone can point to a concrete witness of this text. The work which I described is an Ethiopian composition, made of several pieces, unfortunately no sign of Serapion.

File: 1001-2000/LIT1799Lifeof.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply 1001-2000/LIT1799Lifeof.xml:88]: removed

File: new/LIT6166MiraclesJohnBaptist.xml (hunk 6 above)
Comment (DenisNosnitsin1970) [reply new/LIT6166MiraclesJohnBaptist.xml:222]: Sorry, I can creat "Dossier" but I do not want to have it in the catalogue. The idea of "Hagiographic dossier" is not suitable for a catalogue. In presentation of the content it creats a level that does not exist in the reality (though I can imagine such an entry as point of reference to all text dedicated to the saint, unless the record for a saint is not sutisfactory (why?)). "H dossier" is an overarching comprehensive term that is used in the hagiographic research to refer summarily to all texts related to the saint, major and minor (and their versions). Dossier includes dozens of texts, big and small, independent of parts of other compositions. Normaly dossier has by far more texts than one manuscript contain.

File: 1001-2000/LIT1799Lifeof.xml (hunk 7 above)
Comment (DenisNosnitsin1970) [reply 1001-2000/LIT1799Lifeof.xml:94]: I used to indicate only shelfmarks, this was ok, was it? RQM-029 is not described yet.

File: 1001-2000/LIT1799Lifeof.xml (hunk 8 above)
Comment (DenisNosnitsin1970) [reply 1001-2000/LIT1799Lifeof.xml:91]: ecrm:P129_is_about - correct?

File: 1001-2000/LIT1799Lifeof.xml (hunk 2 above)
Comment (eu-genia) [reply 1001-2000/LIT1799Lifeof.xml:55]: should be DN

File: 1001-2000/LIT1799Lifeof.xml (hunk 7 above)
Comment (eu-genia) [reply 1001-2000/LIT1799Lifeof.xml:94]: it is not at all necessary or requested to have hasAttestation relation for works, as these shall be linked as msItems anyway

what I meant you need relations for the texts which are similar but different to disambiguate

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LIT6166MiraclesJohnBaptist.xml (hunk 6 above)
Comment (eu-genia) [reply new/LIT6166MiraclesJohnBaptist.xml:222]: of course, msItems show which actual parts of a dossier are present in a manuscript, yet it is important to show if a manuscript has e.g. only Gadl, or only Miracles, or a set of texts dedicated to one saint, also to refer to a MS. 

also when we refer to a group ID for the miracles of Mary we do not imply that ALL miracles are in the MS

File: 1001-2000/LIT1799Lifeof.xml (hunk 3 above)
Comment (eu-genia) [reply 1001-2000/LIT1799Lifeof.xml:46]: not added?
should be 
`<langUsage><language ident="en">English</language><language ident="gez">Gǝʿǝz</language></langUsage>`

Comment (DenisNosnitsin1970) [reply 1001-2000/LIT1799Lifeof.xml:46]: I did!

File: 1001-2000/LIT1799Lifeof.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply 1001-2000/LIT1799Lifeof.xml:55]: Sorry, changed.

File: new/LIT6166MiraclesJohnBaptist.xml (hunk 6 above)
Comment (DenisNosnitsin1970) [reply new/LIT6166MiraclesJohnBaptist.xml:222]: "it is important to show if a manuscript has e.g. only Gadl, or only Miracles, or a set of texts dedicated to one saint," - but what for do I prepare a description with works listed as items? If one wants to show what parts of a dossier are included in the manuscript (which is self evident in the description, in my opinion), one needs to carry out a large work to define the dossier, which I do not want to enter now for John the Baptist. And, again, this is not applicable for a catalogue. "Miracles of Mary" is a different matter.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**